src/cpp/shared/game/game.cpp

```diff
@@ -2220,4 +2220,120 @@ void game::ShowMoraleInfo(hero *hro, int dialogType) {
     strcat(gText, "\nnone");
 
   return NormalDialog(gText, dialogType, -1, -1, -1, 0, -1, 0, -1, 0);
+}
+
+void mouseManager::SetPointer(int spriteIdxArg) {
```

Comment:
What changes did you make to this from the original decompiled?

Can you make it more clear?

Reply:
Changed a comment inside it. This is the only thing that changed from original decompiled afair